tests/MakersOfDenmark.Domain.Tests/MODUserTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using MakersOfDenmark.Domain.Enums;
+using MakersOfDenmark.Domain.Models;
+using MakersOfDenmark.Domain.Models.User;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace MakersOfDenmark.Domain.Tests
+{
+    public class MODUserTest
+    {
+        private readonly Fixture _fixture;
+        public MODUserTest()
+        {
+            _fixture = new Fixture();
+        }
+        [Fact]
+        public void CreateUser()
+        {
+            // Arrange
+            var expected = _fixture.Build<MODUser>().Without(x => x.Follows).Without(x => x.Roles).Without(x=> x.CreatedAt).Create();
+
+            // Act
+            var actual = MODUser.CreateUser(expected.FirstName, expected.LastName, expected.Email);
+
+            // Assert
+            actual.Should().BeEquivalentTo(expected, options => options.Excluding(x=>x.Id).Excluding(x=>x.CreatedAt));
```

Comment:
Vi kan gøre brug af 

https://fluentassertions.com/objectgraphs/
"Equivalency Comparison Behavior" her for at teste DateTime med.

Reply:
Efter nærmere eftertanke, hvad er det så vi prøver at teste her? At Fixture kan lave et objekt der minder om vores egen oprettelse af et objekt?